components/x11/xf86-video-vboxvideo/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Aurelien Larcher
+#
+
+X11_CATEGORY=DRIVER
+include ../../../make-rules/shared-macros.mk
+include $(WS_MAKE_RULES)/x11.mk
+
+COMPONENT_NAME= xf86-video-vboxvideo
+COMPONENT_VERSION= 1.0.0
+# Use version string from Xorg module
+IPS_COMPONENT_VERSION= 1.0.1
+COMPONENT_REVISION= 3
```

Comment:
You didn't change the COMPONENT_REVISION here, either.

Reply:
I did but git auto merge decided otherwise..

Reply:
Fixed, thank you!